core/script.runtime/src/kotlin/script/templates/standard/templates.kt

```diff
@@ -16,18 +16,25 @@
 
 package kotlin.script.templates.standard
 
+// discuss
+//
+// These are some 'basic' script templates
+// Should we keep them here?
```

Comment:
The simple variant is easier to grasp by someone who is trying understand the samples or to use the code, no?